content/02.main-text.md

```diff
@@ -255,5 +261,19 @@ Though it is still valuable to have versioned drafts of a review manuscript with
 After releasing the first version of our collaborative review [@doi:10.1101/142760], {{deep_review_post_submission_authors}} new contributors have updated the manuscript (Figure @fig:contrib) and existing authors continue to discuss new literature, [creating a living document](https://github.com/greenelab/deep-review/).
 Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @tag:livecoms].
 
-Our process represents an early step toward open massively collaborative reviews, and there are certainly aspects that can be improved.
-We invite the scientific community to adapt and build upon our experience and open software.
+### The open science movement
```

Comment:
Another option for this header is "Manubot in the context of open science"

Reply:
I like "Manubot in the context of open science"